### PR TITLE
First PR after fork: Adjust build and release

### PR DESCRIPTION
**Pull request by effiban**
I forked this repository in order to overcome [this issue](https://github.com/smoketurner/dropwizard-swagger/issues/206) which broke the Swagger UI in the Dropwizard-Scala template. 
The issue was already fixed (see [PR](https://github.com/smoketurner/dropwizard-swagger/pull/212)) but for some reason the maintainer of the repo hasn't released a new version yet and hasn't responded to any comments.

The problem is only in Scala microservices **for now**, because the Scala template has been migrated to Dropwizard 2 and a newer Swagger version.

This is the first PR after the fork just to setup the repo. Hopefully no more will be needed since the fix is already in the parent.
It contains:
- Removed unnecessary files
- Merged original parent pom into pom of this repository (so we don't have to maintain a forked pom as well)
- Setup Jenkins jobs to replace the original Travis (**WIP**)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,66 +1,61 @@
...
--->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <parent>
-        <groupId>com.smoketurner.dropwizard</groupId>
-        <artifactId>dropwizard-pom</artifactId>
-        <version>2.0.16-1</version>
-    </parent>
```
Comment (effiban) [inline pom.xml:25]: This parent pom includes things we don't need, such as a license plugin which automatically edits the copyright in all of the files and other plugins related to release of an open-source project.
We could also fork the repo of the pom, but that would complicate the maintenance.
So instead I just merged the relevant parts from there into this one.

File: pom.xml [hunk 2]
```diff
@@ -1,66 +1,61 @@
...
 
-    <groupId>com.smoketurner</groupId>
+    <groupId>io.skai</groupId>
     <artifactId>dropwizard-swagger</artifactId>
-    <version>2.0.12-2-SNAPSHOT</version>
+    <version>2.0.${env.BUILD_NUMBER}</version>
```
Comment (tzachz) [inline pom.xml]: maybe we should use `2.0.12.${env.BUILD_NUMBER}` - to preserve the fact that we're forking this specific version and also make sure we don't confuse "real" versions with forked ones.

**Review by tzachz — COMMENTED** [review]
LGTM, had one small comment

File: pom.xml (hunk 2 above)
Comment (effiban) [reply pom.xml]: :+1: changed

**Comment by effiban** [comment]
retest this please

**Comment by effiban** [comment]
retest this please

**Comment by effiban** [comment]
Going to merge so the Jenkins jobs will be created - and I will fix them afterwards

**Pull request merged**